Feature/restaurant

jessicafarias opened the pull request:
In this project, I designed a Website using Webpack.
#I have created 3 functions that helped me to organize my content inside each container.
#The display and classes for each section change after clicking the navbar.
#In order to get all sections, I created a template.html that I called from webpack.config.js using HTMLWebpackPlugin.
#My CSS and images file also have a loader to overwrite everything in a new main.js file inside the dist folder.

jessicafarias committed:
Adding package.json

jessicafarias committed:
Adding webpack

Adding webpack to package.json
Adding webpack cli to package.json
Intall node_modules locally
Adding .gitgnore for node_modules

jessicafarias committed:
Adding index .js and .html

jessicafarias committed:
run npx webpack

Adding main.js in index.html
Adding webpack.config.js file

jessicafarias committed:
Adding watch in module

jessicafarias committed:
Adding bare-bones homepage

jessicafarias committed:
Refactor html to javascript

Convert html to javascript objects

jessicafarias committed:
Fix footer style

jessicafarias committed:
Adding tab-navigation

jessicafarias committed:
Adding hover to navbar

jessicafarias committed:
Import bootstrap and css file using webpack

npm install bootstrap and imported
Install style-loader and css-loader
Adding build in script
Importing images in index.js file.

jessicafarias committed:
Create HTML template

Adding HTMLWebpackPlugin
Create new template.html

jessicafarias committed:
Delete unused folders

jessicafarias committed:
Adding clean webpack plugin

jessicafarias committed:
Update Readme.md

jessicafarias committed:
Fix gotomenu function

rOluochKe reviewed (changes requested):
Hello @jessicafarias 
### You have done well so far :clap: However, you have a few issues to fix:-
- [x] Add linters to the project and ensure they have no errors when a user runs the commands. [Follow this guide for help with adding, installing, and configuring](https://github.com/microverseinc/linters-config/tree/master/javascript)
- [x] We recommend the use of es arrow functions to implement your codebase. See below
```
const unstyle = (parentId, ...styles) => {
    # Your code
}
```
- [x] Add more scripts in the package.json file. Install all the required dependencies, configure and run the commands. See below
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" 
  },
```
### Make all the changes and submit for another code review. Good luck.
- Happy coding!

jessicafarias committed:
Adding linters

Adding stylelint linter
Fixing css files errors
Addding eslint linter
Fixing js files errors

jessicafarias committed:
Implementing arrow functions

jessicafarias committed:
Adding scripts to package.json

Adding webpack-dev-server
Create a automatic local-server in port 8080
Update Readme.md file
Adding dev script
Adding build script
Adding start script

jessicafarias committed:
Fixing eslint errors

jessicafarias commented:
Thank you for your review, it was very helpful.

rOluochKe reviewed (approved):
### You got it right, well done. The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!
